css/dashboard.css

```diff
@@ -0,0 +1,133 @@
+/* 
+ * Plantalytics
+ *     Copyright (c) 2016 Sapphire Becker, Katy Brimm, Scott Ewing, 
+ *       Matt Fraser, Kelly Ledford, Michael Limb, Steven Ngo, Eric Turley.
+ *     This project is licensed under the MIT License.
+ *     Please see the file LICENSE in this distribution for license terms.
+ * Contact: plantalytics.capstone@gmail.com
+ */
+ 
+:root {
+    --page-margin: 10px;
+    --button-size: 40px;
+}
+
+html, body, .map {
+    width: 100%;
+    height: 100%;
+    margin: 0px;
+
+    overflow: hidden;
+}
+
+.map {
+    position: absolute;
+}
+
+.map-overlay {
+    display: none;
+}
+
+/* Button styles */
+.data-buttons {
+    display: inline-block;
+}
+
+.data-button-label > input {
+    visibility: hidden;
+    position: absolute;
+}
+
+.data-button-label > input:checked + .data-button {
+    background-color: lightgray;
```

Comment:
This color change is a bit subtle - maybe change to something a little more obvious?